website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,543 @@
+
+@include 'ui/supported-login-methods-api-docs.mdx'
+
+<Tip>
+
+- To apply a login customization to the `root` namespace, pass an empty string: `namespace_path=""`.
+
+- **Login settings are inheritable by default**. Namespaces inherit login settings from their parent 
+unless inheritance is explicitly disabled. To scope a login setting to a specific namespace, 
+set: `disable_inheritance=true`.
+
+</Tip>
+
+<Tabs>
+<Tab heading="CLI" group="cli">
+
+1. Use `vault write` with the
+   [`sys/config/ui/login/default-auth`](/vault/api-docs/system/config-ui-login-default-auth)
+   endpoint to set the default and/or backup GUI authentication method(s).
+
+    ```shell-session
+    $ vault write sys/config/ui/login/default-auth/<unique_rule_name> \
+      default_auth_type=<default_auth_type>                           \
+      backup_auth_types=<backup_auth_type>                            \
+      backup_auth_types=<backup_auth_type>                            \
+      disable_inheritance=<true_or_false>                             \
+      namespace_path=<namespace_where_the_rule_applies>
+    ```
+
+    For example, to customize the login options for the `dev` namespace and set 
+    `userpass` as the default authentication method with `ldap` and `token` 
+    as additional, backup options:
+
+    <CodeBlockConfig hideClipboard>
+
+    ```shell-session
+    $ vault write sys/config/ui/login/default-auth/dev-gui-login  \
+      default_auth_type=userpass                                  \
+      backup_auth_types=ldap                                      \
+      backup_auth_types=token                                     \
+      disable_inheritance=false                                   \
+      namespace_path=dev
+    ```
+
+    </CodeBlockConfig>
+
+1. Use `vault read` with the
+   [`sys/config/ui/login/default-auth/{rule_name}`](/vault/api-docs/system/config-ui-login-default-auth)
+   endpoint and the new rule name to verify the login configuration. 
+   
+   For example:
+
+    <CodeBlockConfig hideClipboard>
+
+    ```shell-session
+    $ vault read sys/config/ui/login/default-auth/dev-gui-login
+
+    Key                    Value
+    ---                    -----
+    backup_auth_types      [ldap token]
+    default_auth_type      userpass
+    disable_inheritance    false
+    namespace_path         dev/
+    ```
+
+    </CodeBlockConfig>
+
+</Tab>
+<Tab heading="GUI" group="gui">
+
+<Note>
+Login settings can only be viewed or deleted using the Vault GUI. 
+Creating login customizations is only available using the CLI or API to encourage best practices.
+</Note>
+
+- Navigate to the **Settings** section in the Vault GUI sidebar and select **UI Login Settings**.
+- View list of existing login settings.
+- Click on a customization to view more details.
+
+</Tab>
+<Tab heading="API" group="api">
+
+1. Create a JSON file with a new custom login configuration rule that sets your
+   default and/or backup GUI authentication methods:
```

Comment:
I realize `set` is how developers will think about this, but for readability I think `specifies` is the better word.
```suggestion
1. Create a JSON file with a new custom login configuration rule that specifies your
   default and/or backup GUI authentication methods:
```

Reply:
Of course we use set a lot throughout, so I could be persuaded either way.